lib/src/view/helper/chucker_ui_helper.dart

```diff
@@ -34,7 +34,8 @@ class ChuckerUiHelper {
     required DateTime requestTime,
   }) {
     if (ChuckerUiHelper.settings.showNotification &&
```

Comment:
@cscipher As you can see there is already a property `ChuckerUiHelper.settings.showNotification` for the same purpose.